src/main/resources/templates/login.html

```diff
@@ -1,68 +1,68 @@
-                       id="password"
-                       th:name="password"
-                       class="form-control"/>
-                <label class="form-label" for="password">Password</label>
-            </div>
+        <hr>
+        <!-- Email input -->
+        <div class="form-floating ">
+            <input type="text"
+                   id="usernameOrEmail"
+                   th:name="usernameOrEmail"
+                   placeholder="Username or email address"
+                   class="form-control"/>
+            <label class="form-label" for="usernameOrEmail">Username or email address</label>
+        </div>
 
-            <!-- 2 column grid layout for inline styling -->
-            <div class="row mb-4">
-                <div class="col d-flex justify-content-center">
-                    <!-- Checkbox -->
-                    <div class="form-check">
-                        <input class="form-check-input" type="checkbox" value="" id="form2Example31" checked/>
-                        <label class="form-check-label" for="form2Example31"> Remember me </label>
-                    </div>
-                </div>
+        <!-- Password input -->
+        <div class="form-floating ">
+            <input type="password"
+                   id="password"
+                   th:name="password"
+                   placeholder="Password"
+                   class="form-control"/>
+            <label class="form-label" for="password">Password</label>
+        </div>
 
-                <div class="col">
-                    <!-- Simple link -->
-                    <a href="#">Forgot password?</a>
+        <!-- 2 column grid layout for inline styling -->
+        <div class="row mt-3">
+            <div class="col d-flex justify-content-center">
+                <!-- Checkbox -->
+                <div class="form-check">
+                    <input class="form-check-input" type="checkbox" value="" id="form2Example31" checked/>
+                    <label class="form-check-label" for="form2Example31"> Remember me </label>
                 </div>
             </div>
 
-            <div class="btn-toolbar">
-                <!-- Submit button -->
-                <div class="btn-group">
-                    <button type="submit" class="btn btn-primary btn-block mb-4">Sign in</button>
-                    <a th:href="@{/auth/register}" class="btn btn-success btn-block mb-4">New account</a>
-                </div>
+            <div class="col">
+                <!-- Simple link -->
+                <a href="#" class="link-warning text-decoration-none">Forgot password?</a>
             </div>
+        </div>
+
+        <div class="d-flex align-items-center justify-content-start mt-5">
+            <!-- Submit button -->
+            <button type="submit" class="btn btn-primary btn-block">Sign in</button>
+            <a th:href="@{/auth/register}" class="link-warning text-decoration-none ms-4">New account</a>
```

Comment:
The color is okay but it might look better slightly more transparent or a shade darker.

Reply:
For responsive design principles, we don't have to use so many colors for the buttons/ links.
If we make it look like a rainbow, it will create confusions for the user
![login form](https://user-images.githubusercontent.com/44172060/235291805-f9f7ef74-3b93-42fc-81d5-d4e826a83b51.png)